cache/memberlist.go

```diff
@@ -58,7 +59,7 @@ func (c *Cache) runMemberList(ctx context.Context) error {
 		mh.log.Error().Msg("multiple cache servers not supported")
 	}
 	<-ctx.Done()
-	mh.memberlist.Leave()
+	mh.memberlist.Leave(1 * time.Second) //nolint
```

Comment:
This error should be handled. Maybe:

```
if err := mh.memberlist.Leave(time.Second); err != nil {
		mh.log.Error().Err(err).Msg("could not broadcast leave message")
}
```

Reply:
You're right.  Given there's not much to do here, I didn't bother.  But we should put in a log message.